eval/src/vespa/eval/eval/fast_sparse_map.h

```diff
@@ -148,7 +147,7 @@ class FastSparseMap
     void each_map_entry(F &&f) const {
         _map.for_each([&](const auto &entry)
                       {
-                          f(entry.first.start, entry.second, entry.first.hash);
+                          f(entry.second * _num_dims, entry.second, entry.first.hash);
```

Comment:
an alternative is to not pass start and redefine the make_addr function instead to get an address from a subspace directly. That would trade a divide with a multiply for the iterate case and we would save the multiply for the time we actually need the address.